main.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-plusplus */
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix linter errors without this kind of comment `/* eslint-disable ... */` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃

Reply:
Hello @mukhammadrizooff 
I know some linters are easy to solve, but if you see, the linters disabled are related to the tip Microverse gave us at the first project of the week:
![image](https://user-images.githubusercontent.com/84760694/235009556-ae1c8ec7-a2f0-4cd1-b2dc-7eb757f62ac8.png)
As you can see, they say we can disable multiple classes linter.
On the other hand, can you specify a good reason why shouldn't I disable the "no-plus-plus" linter? With a good reason I will remove it in all my next projects, but if not, I'll use it anyways, as It increases my work experience and skills already :)

Reply:
> Hello @mukhammadrizooff I know some linters are easy to solve, but if you see, the linters disabled are related to the tip Microverse gave us at the first project of the week: ![image](https://user-images.githubusercontent.com/84760694/235009556-ae1c8ec7-a2f0-4cd1-b2dc-7eb757f62ac8.png) As you can see, they say we can disable multiple classes linter. On the other hand, can you specify a good reason why shouldn't I disable the "no-plus-plus" linter? With a good reason I will remove it in all my next projects, but if not, I'll use it anyways, as It increases my work experience and skills already :)

Hi @EmiLoKiJu, This `Tip` for the previous milestone's requirements, in this milestone you should implement [this requirement guide](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website_v1_1.md) You should use a separate requirements guide for each milestone 👍🏼

Reply:
⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️ ⬆️